From e310f3266d80d5142e529a8f03a6782198703ade Mon Sep 17 00:00:00 2001
From: George Zacharia <george.zcharia@gmail.com>
Date: Wed, 22 Jan 2020 12:50:16 +0530
Subject: [PATCH] Allow only two lines in the list

More than 2 lines will looks bad on the layout while 1 line won't give all the details
---
 .../moez/QKSMS/feature/conversations/ConversationsAdapter.kt    | 2 +-
 presentation/src/main/res/layout/conversation_list_item.xml     | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/presentation/src/main/java/com/moez/QKSMS/feature/conversations/ConversationsAdapter.kt b/presentation/src/main/java/com/moez/QKSMS/feature/conversations/ConversationsAdapter.kt
index c2c9e7be6..608f7c91b 100644
--- a/presentation/src/main/java/com/moez/QKSMS/feature/conversations/ConversationsAdapter.kt
+++ b/presentation/src/main/java/com/moez/QKSMS/feature/conversations/ConversationsAdapter.kt
@@ -58,7 +58,7 @@ class ConversationsAdapter @Inject constructor(
 
                 binding.snippet.setTypeface(binding.snippet.typeface, Typeface.BOLD)
                 binding.snippet.setTextColor(textColorPrimary)
-                binding.snippet.maxLines = 5
+                binding.snippet.maxLines = 2 // Just in case,else breaks the view
 
                 binding.unread.isVisible = true
 
diff --git a/presentation/src/main/res/layout/conversation_list_item.xml b/presentation/src/main/res/layout/conversation_list_item.xml
index 1ed7eebfc..2cf5f642b 100644
--- a/presentation/src/main/res/layout/conversation_list_item.xml
+++ b/presentation/src/main/res/layout/conversation_list_item.xml
@@ -63,7 +63,7 @@
         android:layout_height="wrap_content"
         android:layout_marginTop="2dp"
         android:ellipsize="end"
-        android:maxLines="1"
+        android:maxLines="2"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toStartOf="@id/pinned"
         app:layout_constraintStart_toStartOf="@id/title"
